pmpro-failed-payment-limit.php

```diff
@@ -62,7 +62,7 @@ function pmprofpl_pmpro_subscription_payment_failed($order) {
 		if($worked === true) {							
 			//send an email to the member
 			$myemail = new PMProEmail();
-			$myemail->sendCancelEmail($user->ID);
+			$myemail->sendCancelEmail($user, $old_level_id);

```

Comment:
Although it would make sense to pass the specific level that had failed payments to the email, as written, the prior code will cancel ALL levels if a single level reaches the failed payment limit. Therefore on MMPU setups, only passing the level that failed to the `sendCancelEmail()` method would be misleading.
```suggestion
			$myemail->sendCancelEmail($user);

```